build/common.sh

```diff
@@ -159,19 +159,21 @@ function _feedback() {
 
   local timestamp
   timestamp="$(date --iso-8601=seconds --utc)"
-  # Use the provided color code label
-  local color
-  color="${1}"
+
+  local log_level
+  log_level="${LOG_LEVEL:-WARNING}"
```

Comment:
This sets an explicit default to `WARNING` inside the `common.sh` scope where it's used.  I am avoiding adding this as an `ARG`/`ENV` unless we need it scoped more broadly than just `common.sh`.